content/en/admin/optional/object-storage.md

```diff
@@ -121,6 +121,15 @@ Default: 5 (seconds)
 
 The number of seconds before the HTTP handler should timeout while waiting for an HTTP response.
 
+#### `S3_RETRY_LIMIT`
+
+Adjusting this may be needed for some S3 providers, particularly when there are connectivity issues between the Mastodon server and the S3 storage. This may be observed when posts from other instances have missing media.
+
+Defaults to `0`, not suggested to increase beyond `3` due to database performance concerns.
```

Comment:
This is a nitpick, but it's not really about database performance but jobs holding up workers for longer than necessary (so work queues growing bigger).